scripts/zf_mkdist

```diff
@@ -33,9 +33,11 @@ usage() {
     echo "(Installed onload & onload-devel utilised when onload_tarball omitted.)"
     echo
     echo "options:"
-    echo "  --version <version>"
-    echo "  --shim"
-    echo "  --name <tarball_prefix>"
+    echo "  --version <version>    - Specify version string for tarball."
+    echo "  --shim                  - Build the socket shim library. For testing only."
+    echo "                            Requires onload_tarball"
+    echo "  --debug                 - Build debug version of tcpdirect."
+    echo "  --name <tarball_prefix> - Specify prefix for generated tarball. Default is \"zf\"."
```

Comment:
Yeah, runbench uses `zf_mkdist` to build tcpdirect and assumes that the tarball of binaries begins with "zf". `zf_make_tarball` is more designed for the packaging workflow where the default prefix is "tcpdirect" rather than "zf"